runtimes/ruby.nix

```diff
@@ -0,0 +1,57 @@
+{pkgs, piston, ...}:
+let 
+    pkg = pkgs.ruby_3_1;
```

Comment:
any reason to use exactly ruby_3_1?

Reply:
it's the latest ruby. otherwise, it would pull 2.7.5 from the 'ruby' nixpkg